src/sagemaker/djl_inference/model.py

```diff
@@ -924,3 +1196,60 @@ def __init__(
                 "number_of_partitions will be ignored"
             )
         self.number_of_partitions = tensor_parallel_degree or self.number_of_partitions
+
+    def partition(
+        self,
+        instance_type: str,
+        s3_output_uri: str = None,
+        job_name: Optional[str] = None,
+        volume_kms_key: Optional[str] = None,
+        output_kms_key: Optional[str] = None,
+        use_spot_instances: bool = False,
+        max_wait: int = None,
+        enable_network_isolation: bool = False,
+    ):
+        """Partitions the model using SageMaker Training Job. This is a synchronous API call.
+
+        Args:
+            instance_type (str): The EC2 instance type to partition this Model.
+                    For example, 'ml.p4d.24xlarge'.
+            s3_output_uri (str): S3 location for saving the training result (model
+                    artifacts and output files). If not specified, results are
+                    stored to a default bucket. If the bucket with the specific name
+                    does not exist, it will be created.
+            job_name (str): Training job name. If not specified, a unique training job
+                        name will be created.
+            volume_kms_key (str): Optional. KMS key ID for encrypting EBS
+                volume attached to the training instance (default: None).
+            output_kms_key (str): Optional. KMS key ID for encrypting the
+                training output (default: None).
+            use_spot_instances (bool): Specifies whether to use SageMaker
+                Managed Spot instances for training. If enabled then the
+                ``max_wait`` arg should also be set.
+
+                More information:
+                https://docs.aws.amazon.com/sagemaker/latest/dg/model-managed-spot-training.html
+                (default: ``False``).
+            max_wait (int): Timeout in seconds waiting for spot training
+                job (default: None). After this amount of time Amazon
+                SageMaker will stop waiting for managed spot training job to
+                complete (default: None).
+            enable_network_isolation (bool): Specifies whether container will
+                run in network isolation mode (default: ``False``). Network
+                isolation mode restricts the container access to outside networks
+                (such as the Internet). The container does not make any inbound or
+                outbound network calls. Also known as Internet-free mode.
+        Returns:
+            None
+        """
+
+        super(FasterTransformerModel, self).partition(
+            instance_type,
+            s3_output_uri,
+            job_name,
+            volume_kms_key=volume_kms_key,
+            output_kms_key=output_kms_key,
+            use_spot_instances=use_spot_instances,
+            max_wait=max_wait,
+            enable_network_isolation=enable_network_isolation,
+        )
```

Comment:
We don't need this, I don't see any FT specific logic. Parent implementation should be enough.

Reply:
Okay

Reply:
Resolved